core/src/main/java/org/infinispan/stats/impl/CacheContainerStatsImpl.java

```diff
@@ -64,7 +64,11 @@ public void resetStatistics() {
       if (getStatisticsEnabled()) {
          for (String cn : cm.getCacheNames()) {
             if (cm.cacheExists(cn)) {
-               cm.getCache(cn).getAdvancedCache().getStats().reset();
+               try {
+                  cm.getCache(cn).getAdvancedCache().getStats().reset();
+               } catch (SecurityException e) {
+                  // no-op, skipping this cache
```

Comment:
I see two possibilities how to avoid it:
* check, if the user has access to the cache before accessing it, but AFAICT in case user has access to it, it would be checked twice (before accessing the cache and during accessing the cache), so this wouldn't be very efficient approach (especially when assuming that in most of the cases user has correct permissions and has access to the case)
* use `SecurityActions` to access the cache without checks, but my understanding is that also statistics should respect security setup and shouldn't access caches without required permissions (i.e. IMHO this is not an option at all, but maybe I'm wrong)